test/utils/constants.ts

```diff
@@ -1 +1,7 @@
 export const DEFAULT_PASSWORD = 'default-password';
+export const PRIVATE_GENERATOR_SEED =
+  'waves private node seed with waves tokens';
```

Comment:
It's mentioned in docs as "default miner account with all Waves tokens" (https://github.com/wavesplatform/private-node-docker-image#configuration-details), may be name it something like `DEFAULT_MINER_SEED`? Adding a link to docs would be helpful also.